packaging/templates/mainnet-v1/sentry/sentry/bor/pbss_config.toml

```diff
@@ -1,15 +1,16 @@
-chain = "mumbai"
+chain = "mainnet"
 # identity = "node_name"
 # verbosity = 3
 # vmdebug = false
 datadir = "/var/lib/bor/data"
 # ancient = ""
-# "db.engine" = "leveldb"
+db.engine = "pebble"
+state.scheme = "path"
 # keystore = ""
 # "rpc.batchlimit" = 100
 # "rpc.returndatalimit" = 100000
 syncmode = "full"
-gcmode = "archive"
+# gcmode = "full"
```

Comment:
Can we add a comment here that pbss doesn't support archive mode, just to avoid any confusions and inapt config in the future.

Reply:
Updated and added